package/base-files/files/bin/config_generate

```diff
@@ -42,8 +43,7 @@ generate_static_network() {
 		set network.loopback='interface'
 		set network.loopback.device='lo'
 		set network.loopback.proto='static'
-		set network.loopback.ipaddr='127.0.0.1'
-		set network.loopback.netmask='255.0.0.0'
+		add_list network.loopback.ipaddr='127.0.0.1/8'
```

Comment:
Why the change to `add_list`?

Reply:
https://github.com/openwrt/openwrt/pull/13780#issuecomment-1778206487